node/bin/src/prover_api/prover_server/v1/models.rs

```diff
@@ -0,0 +1,52 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize)]
+pub(super) struct BatchDataPayload {
+    pub block_number: u64,
```

Comment:
this is batch_number, no? I was thinking we can rename now since we introduce a new interface version